src/main/kotlin/com/amazon/opendistroforelasticsearch/indexmanagement/transform/settings/TransformSettings.kt

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright 2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+package com.amazon.opendistroforelasticsearch.indexmanagement.transform.settings
+
+import org.elasticsearch.common.settings.Setting
+import org.elasticsearch.common.unit.TimeValue
+
+class TransformSettings {
+
+    companion object {
+        const val DEFAULT_RENEW_LOCK_RETRY_COUNT = 3
+        const val DEFAULT_RENEW_LOCK_RETRY_DELAY = 1000L
+
+        val TRANSFORM_JOB_SEARCH_BACKOFF_COUNT: Setting<Int> = Setting.intSetting(
+            "opendistro.transform.internal.search.backoff_count",
```

Comment:
thought it would imply that these are internal settings for transform job, but not particular about it. Can remove if it makes it confusing. 

My reason is to avoid 
``transform.search.backoff_count `` is to avoid implying settings for searching transformed indices

Reply:
Got it, that's fine